banzai/tests/e2e-k8s.yaml

```diff
@@ -201,10 +203,10 @@ spec:
       resources:
         requests:
           cpu: 0.1
-          memory: 512M
+          memory: 1Gi
         limits:
           cpu: 1
-          memory: 1Gi
+          memory: 2Gi
```

Comment:
I upped this because in the e2e tests we kick off reduction by shelling into the banzai-listener container. This causes a process in the container to read a set of BPMs and now readnoise maps off disk into memory temporarily to seed the DB with them and we were getting OOM issues. Seemed appropriate as we're adding more data to these e2e tests.